fastlane/Fastfile

```diff
@@ -125,32 +125,22 @@ platform :android do
     UI.verbose("Deploying #{version}")
     gradle(
       tasks: [
-        "androidSourcesJar", "androidJavadocJar", "publish --no-daemon --no-parallel"
+        "publish --no-daemon --no-parallel"
       ],
       properties: gradleProperties
     )
 
     UI.verbose("Creating special version for Unity IAP with BillingClient 4 and Amazon 2 for version: #{version}")
-    gradleProperties["PUBLISH_VARIANT"] = "unityIAPRelease"
+    gradleProperties["ANDROID_VARIANT_TO_PUBLISH"] = "unityIAPRelease"
     
     UI.verbose("Adding -unityiap to artifact ids")
 
     gradle(
       tasks: [
-        "androidSourcesJar", "androidJavadocJar", "publish --no-daemon --no-parallel"
+        "publish --no-daemon --no-parallel"
       ],
       properties: gradleProperties
     )
-    
-    unless is_snapshot_version?(version)
```

Comment:
I tried publishing a snapshot and it doesn't do `closeAndReleaseRepository` anyways so this is safe to remove